english/total-war/universe/universe.md

```diff
@@ -0,0 +1 @@
+Warhammer Fantasy is a fantasy setting, created by Games Workshop, which is used by many games, including the table top wargame Warhammer Fantasy Battle, the Warhammer Fantasy Roleplay (WFRP) pen-and-paper role-playing game, and the MMORPG Warhammer Online: Age of Reckoning. More info [here](https://en.wikipedia.org/wiki/Warhammer_Fantasy_(setting)).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.